packages/@react-spectrum/tag/stories/TagGroup.stories.tsx

```diff
@@ -122,6 +122,20 @@ export const WithIcons: TagGroupStory = {
   )
 };
 
+export const WithSomeIcons: TagGroupStory = {
```

Comment:
instead of a new one, can you change an existing story and just remove/add an avatar? That way we can also check chromatic.